pkg/webhook/patch.go

```diff
@@ -849,3 +912,26 @@ func addShareProcessNamespace(pod *corev1.Pod, app *v1beta2.SparkApplication) *p
 	}
 	return &patchOperation{Op: "add", Path: "/spec/shareProcessNamespace", Value: *shareProcessNamespace}
 }
+
+// SubPath support
+func addConfigMapVolumeMountSubPath(pod *corev1.Pod, configMapVolumeName string, mountPath string, subPath string) *patchOperation {
+	mount := corev1.VolumeMount{
+		Name:      configMapVolumeName,
+		ReadOnly:  true,
+		MountPath: mountPath,
+		SubPath:   subPath,
+	}
+	return addVolumeMount(pod, mount)
+}
+
+func findVolumeMountIndex(container *corev1.Container) int {
+	// Find the driver or executor container in the pod.
```

Comment:
Same as the last private method comment - if this line is a comment we want to keep add it to the method itself